.eslintrc.js

```diff
@@ -207,6 +207,63 @@ module.exports = {
             ignoreRestSiblings: true,
           },
         ],
+        '@typescript-eslint/naming-convention': [
```

Comment:
This is an identical copy of latest `@typescript-eslint/naming-convention` rule (not released yet). And it will be removed once https://github.com/MetaMask/eslint-config/pull/388 is added to new release and we update the `@metamask/eslint-config` package to 14.x.0 version

Reply:
We should record this context in the file as well:

```suggestion
        // This rule temporarily applies the latest `@typescript-eslint/naming-convention` config found in `@metamask/eslint-config`.
        // TODO: Remove once `@metamask/eslint-config` is updated to `^14.0.0`.
        '@typescript-eslint/naming-convention': [
```